woocommerce/woocommerce
# Update/my subscriptions wp installer

### Changes proposed in this Pull Request:

This PR is part of the My subscriptions feature branch.

It adds support for using the WordPress installer in My subscriptions.
For the installer to work this PR had to add download paths to the info request for themes and plugins.

Because there isn't a WordPress API endpoint for activating plugins and themes, we had to add one to the marketplace API.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open the [My subscriptions page](http://localhost:8888/wp-admin/admin.php?page=wc-admin&tab=my-subscriptions&path=%2Fextensions) 
2. Connect to local Woo.com (branch `add/product-slug-to-helper-subscriptions`)
3. Install a WCCOM extensions, .org extension, and a theme
4. Confirm that installing works



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Use WP installer for my subscriptions instead of the Woo installer API.

</details>


https://github.com/woocommerce/woocommerce/assets/1199991/b7399709-ba08-4c9b-b1b1-0ef62d226925
